core/tests/protractor_desktop/explorationHistoryTab.js

```diff
@@ -138,94 +138,98 @@ describe('Exploration history', function() {
         highlighted: true
       },
       12: {
-        text: '    feedback:',
+        text: '    dest_if_really_stuck: null',
```

Comment:
Why are we adding `dest_if_really_stuck` in field number 12 specifically? Why can we not just add it at the end?

Reply:
Just to preserve the general order we've maintained everywhere. For eg, line 477 in the same file.